samples/nrf9160/cloud_simple/README.rst

```diff
@@ -0,0 +1,110 @@
+
+.. note::
+   The nRF9160 DK and Thingy:91 comes stock with certificates required for connection to NRF Cloud. For NRF Cloud no extra setup steps are required to use the ``cloud_simple`` sample. See `NRF Cloud web interface <https://nrfcloud.com/#/>`_ for more information.
+
+
+Configurations
+**************
+
+The configurations for the sample are located in ``cloud_simple/src/prj.conf``
+
+ - ``CONFIG_CLOUD_BACKEND`` <- Determines the cloud backend to be used.
+ - ``CONFIG_CLOUD_PUBLICATION_SEQUENTIAL`` <- Publish message to cloud sequentially.
+ - ``CONFIG_CLOUD_PUBLICATION_BUTTON`` <- Publish message to cloud upon a button press.
+ - ``CONFIG_CLOUD_MESSAGE`` <- Modifies the message published to the cloud service.
+ - ``CONFIG_CLOUD_MESSAGE_PUBLICATION_INTERVAL`` <- Modifies how often the message should be published to the cloud service.
+
+.. note::
+   To output data in the terminal window located in the `NRF Cloud web interface <https://nrfcloud.com/#/>`_ the data format must be in JSON format.
+
+Functionality and Supported Technologies
+****************************************
+
+The communicaton protocol supported by the sample is dependent on the cloud backend that is used. Please see the documentation for the individual cloud backend libraries.
+
+Functions
+=========
+
+:cpp:func:`cloud_get_binding()`     : Binds to a desired cloud backend using a identification string.
+
+:cpp:func:`cloud_init()`            : Initializes the the cloud connection.
+
+:cpp:func:`cloud_connect()`         : Connects to the cloud service.
+
+:cpp:func:`cloud_ping()`            : Pings the cloud service, should be called periodically.
+
+:cpp:func:`cloud_input()`           : Retrieve data from the cloud service.
+
+:cpp:func:`cloud_send()`            : Send data to the cloud service.
+
+Cloud events used in the sample
+===============================
+
+:cpp:enum:`CLOUD_EVT_CONNECTED`               : Connected to the cloud service.
+
+:cpp:enum:`CLOUD_EVT_READY`                   : Cloud connection established and ready to be communicated with.
+
+:cpp:enum:`CLOUD_EVT_DISCONNECTED`            : Disconnected from the cloud service.
+
+:cpp:enum:`CLOUD_EVT_DATA_RECEIVED`           : Receive data from the cloud service.
+
+.. note::
+   Not all functionality present in the generic cloud API are used by the different cloud backends.
+
+Requirements
+************
+
+* One of the following development boards:
+
+ * nRF9160 Development Kit board (PCA_20035)(Thingy:91)
+ * nRF9160 Development Kit board (PCA_10090)(nRF9160 DK)
+
+
+Building and running
+********************
+
+The sample can be found under ``samples/cloud_simple`` in the nRF Connect SDK folder structure.
```

Comment:
```suggestion
.. |sample path| replace:: :file:`samples/nrf9160/cloud_simple`
```
if we are changing the name of the sample to cloud_client, please remember to update here as well.